### DISCUSSION TITLE
MDP for Hassio, Database question

### DISCUSSION THREAD

**Discussion by NicMcBrain** [Poeschl/Hassio-Addons#151] [Q&A]
Hi,

I'm new to Hassio (and the whole raspberry world). I installed it on a raspberry pi 4-8go, and I was looking for a way to play music locally (directly through the raspberry-pi's jack output), as it's hooked up to my amplifier.
Found your add-on: it's exactly what I was looking for (thank you for that), so I installed it.
It seems to be working, but in the ympd / Browse Database, it's pointing to root/.

Is there a way to change it? (to /media or something else?)

Thanks in advance

**Comment by Poeschl** [comment]
Hm, let me see. In the default config the root folder is indeed the `/share/mpd/music` folder. But I mapped the media folder as well. So a custom config might fix it for you.

OR

Give me some time ;) Will look into a setting to specify the root folder.

**Reply by NicMcBrain** [reply]
Thank you for your reply.
I added a music file and folder in the /share/mpd/music, and I also have some in the /media folder... still I can't see them in the ympd...
Your suggestion to specify a root folder is looking better and better ;)
Joke appart, it'd certainly be a nice upgrade, easy to work with.

**Comment by NicMcBrain** [comment]
By the way, where is this configuration file, I can't seem to find it... (though I must say I don't exactly know where to look)...
Looked through samba: nothing there
Same for File editor: nothing there
Thanks again in advance

**Reply by Poeschl** [reply]
As starting point the [default configuration of the addon](https://github.com/Poeschl/Hassio-Addons/blob/master/mpd/root/etc/mpd.conf) can be used. Place it on /media or /share and specify the path of the custom config with the `custom_config` setting.

**Reply by NicMcBrain** [reply]
Thanks a lot, works like a charm.
Out of curiosity, how do you call back a saved playlist?
I see them in the folder, but I don't see anything in the GUI to call them back.
Thanks again (and sorry for my ignorance :))

**Reply by NicMcBrain** [reply]
Found it, in the database.
Another question: does the db updates itself? How?

**Reply by Poeschl** [reply]
I think the database watches for file changes, if configured. For myself I used the "Update DB" button on ympd or put an automation on HA to do it periodically.

**Reply by NicMcBrain** [reply]
Excellent, thanks a lot
